src/main/java/org/repodriller/persistence/csv/CSVFile.java

```diff
@@ -47,11 +48,13 @@ public CSVFile(String path, String name, boolean append) {
 
 	@Override
 	public synchronized void write(Object... line) {
-		
+		String field;
```

Comment:
this variable can be put inside the for loop, right? Then, we make sure no one is gonna use it afterwards!